test/integration/features/story_registry.feature

```diff
@@ -0,0 +1,42 @@
+@linux
+Feature: Local image to image-registry to deployment
+
+    The user creates a local container image with an app. They then
+    push it to the openshift image-registry in their
+    project/namespace. They deploy and expose the app and check its
+    accessibility.
+
+    Scenario Outline: Start CRC
+        Given executing "crc setup" succeeds
+        When starting CRC with default bundle and hypervisor "<vm-driver>" succeeds
+        Then stdout should contain "CodeReady Containers instance is running"
+        And executing "eval $(crc oc-env)" succeeds
+        And login to the oc cluster succeeds
```

Comment:
This is still not a full proof and might not be consistent on successive test run since there is still chance that Auth might not working. so I was hoping to have `crc status` check also and may be put in retry for 2 mins.